lib/model/migrations/20230406-01-add-entity-def-fields.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = async (db) => {
+  // Add columns
+  await db.raw(`
+ALTER TABLE entity_defs
+ADD COLUMN "creatorId" integer,
+ADD COLUMN "userAgent" varchar(255)
+`);
+
+  // Assign submission submitter id to entity def creator id
+  await db.raw(`
+UPDATE entity_defs
+set "creatorId" = submission_defs."submitterId",
+"userAgent" = submission_defs."userAgent"
+FROM submission_defs
+WHERE entity_defs."submissionDefId" = submission_defs.id;
+  `);
+};
+
```

Comment:
We have to remove `submissionDefId` as well

Reply:
Let's leave it for now - we can take it out when we have a better plan for entity source.